dhall/src/Dhall/Diff.hs

```diff
@@ -576,6 +578,7 @@ skeleton (Union {}) =
     <>  ignore
     <>  " "
     <>  rangle
+    <> " (a union type)"
```

Comment:
Should this be "a union"? It's talking about an instance of a union type, I think?

Reply:
@timbertson: No, this is a union type.  A union value is of the form `UnionType.Alternative x`